### ISSUE TITLE
Improve handling of command line arguments

### ISSUE THREAD

**Issue by nvbn** [nvbn/thefuck#620]
We have 4 cases:

* non-configured `fuck` call should ignore command line arguments;
* `thefuck` call should handle command line arguments if the first argument starts with `-`;
* `fuck` alias should have some generated placeholder in the end and handle arguments only after it;
* `fuck` alias without placeholder should ignore arguments.

**Comment by nvbn** [comment]
Now with zsh and bash we can do:

```bash
➜  git br
git: 'br' is not a git command. See 'git --help'.

Did you mean one of these?
	branch
	var
➜ fuck -y
git branch
* 620-command-line-arguments
➜ fuck -v
The Fuck 3.15 using Python 3.5.1+
➜ fuck --help
usage: thefuck [-v] [-a [ALIAS]] [-h] [-y] [command [command ...]]

positional arguments:
  command               command that should be fixed

optional arguments:
  -v, --version         show program's version number and exit
  -a [ALIAS], --alias [ALIAS]
                        [custom-alias-name] prints alias for current shell
  -h, --help            show this help message and exit
  -y, --yes             execute fixed command without confirmation
```

**Comment by nvbn** [comment]
Implemented in 3.16.
